packages/typescript-estree/src/convert.ts

```diff
@@ -1167,12 +1167,30 @@ export class Converter {
 
       case SyntaxKind.PropertyDeclaration: {
         const isAbstract = hasModifier(SyntaxKind.AbstractKeyword, node);
+        const isAccessor = hasModifier(SyntaxKind.AccessorKeyword, node);
+
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type -- TODO - add ignore IIFE option
+        const type = (() => {
+          if (isAccessor) {
+            if (isAbstract) {
+              return AST_NODE_TYPES.TSAbstractAccessorProperty;
```

Comment:
I'm confused why codecov thinks this isn't covered by tests. I can debug it locally to see the `TSAbstractAccessorProperty` and `AccessorProperty` cases being hit.

Reply:
it might be because the tests are in `ast-spec` instead of in here?
maybe we need to adjust how the coverage is collected?

Reply:
Filed https://github.com/typescript-eslint/typescript-eslint/issues/6116 🤷